docs/apis-clients/go-client/index.md

```diff
@@ -2,7 +2,7 @@
 id: index
 title: "Go client"
 sidebar_label: "Quick reference"
-description: "Here, we'll show you how to instantiate the client."
+description: "Here, we'll show you how to instantiate the client in a Go application to interact with Camunda Platform 8."
```

Comment:
Note: This feedback is non-blocking! We could totally iterate on this.

I think what we've done for many of these descriptions is extracted the first sentence or two from the page, but I think we should put more marketing/SEO-type descriptions in each of these pages. As far as I'm aware, this field is used on two places: 1. the `meta name="description"` tag on the page, which is used by search engines to identify content, and 2. The DocCards in this PR, to show the card description. In both cases, I think the description should capture the essence of the page, which I think some of them don't.

This specific one, for example, seems like it describes a specific step in a tutorial. That's not really what the page is about -- it describes a client for interacting programmatically with Camunda, written in Go.

I'm not going to comment on each individual description, but I agree with your point in the body of this PR about making them shorter, so that more meaningful information shows up in the cards....and in doing that, I think they should also be distilled to describe why a person might come to this page, rather than be a snippet of the content on this page.